### ISSUE TITLE
specfile mca module not loading last data

### ISSUE THREAD

**Issue by physwkim** [silx-kit/silx#3611]
I've noticed that SPEC's array_dump function puts a trailing space at the end of the mca array.
As a brief example, if there is a space before line-feed, the last data is not read.

```
#S 1 mythen_save Point:0 Scan:1 Datafile:/temp.dat
#T 5 (Seconds)
#@MCA %25C
#@CTIME 5
#I 0.000000
@A 253 253 253 279 284 286 
```
There are 6 data, but only 5 are actually loaded.
```
from silx.io.specfile import SpecFile
sf = SpecFile('test.dat')
sf[0].mca[0]

Out: array([253., 253., 253., 279., 284.])
```
If there is no trailing space at the end of the data file, it load normally.

**Comment by llohse** [comment]
I ran into the same problem. 

+1 for fixing this

**Comment by vasole** [comment]
closed by #3612
